### ISSUE TITLE
night not working on mac

### ISSUE THREAD

**Issue by flypenguin** [sonnie-sonnig/ia_typora#1]
ia typora night does not seem to work on mac (it's not dark). i know you said it's untested, just wanted to let you know. cheers for the theme, still!

**Comment by sonnie-sonnig** [comment]
Yeah, I already heard it. Unfortunately I have no chance to test it or any idea, why this does not work. If you have any ideas, I'd try to implement it. Or you can fork it. :)

**Comment by herabit** [comment]
I figured out the issue, it has to do with file names for some reason. Change the underscores to dashes and it should resolve the issue.

**Comment by sonnie-sonnig** [comment]
Awesome, TY! 
Maybe it has something to do with the Mac-filesystem.
I'll comment this in the README.

**Comment by herabit** [comment]
I made a pull request that resolves the issue.
